server/src/controllers/Events.js

```diff
@@ -11,17 +11,17 @@ const Event = require('../models/Event');
 module.exports = {
   async getEvents(_req, res) {
     const events = await Event.find();
-    res.json(events);
+    return res.status(200).json(events);
   },
   async getEventById(req, res) {
     try {
       const eventId = req.params.id;
       const event = await Event.findById(eventId);
       res.json(event);
     } catch (err) {
-      res.status(203).send({
-        err: err
-      });
+      return res.status(400).json({
+        message: err.message
+      })
```

Comment:
Since 400 is a bad request, it'd be great if you could replace the key with "err" or "error" to signify that this is, in fact, an error

Reply:
Noted